src/aux-server/aux-web/aux-player/scene/PlayerGame.ts

```diff
@@ -39,6 +39,7 @@ import {
 import { WebVRDisplays } from '../../shared/WebVRDisplays';
 import { Subject } from 'rxjs';
 import { MenuItem } from '../MenuContext';
+import { CameraRigControls } from 'aux-web/shared/interaction/CameraRigControls';
```

Comment:
@shaneYeti Can you make this import use a relative path instead of the auto-imported path?

Reply:
made this a relative path